easybuild/easyconfigs/o/OpenMM/OpenMM-8.0.0-foss-2023a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,66 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# Update to 7.5.1
+# J. Sassmannshausen / GSTT
+
+easyblock = 'CMakeMake'
+
+name = 'OpenMM'
+version = '8.0.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://openmm.org'
+description = "OpenMM is a toolkit for molecular simulation."
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'opt': True}
+
+source_urls = ['https://github.com/openmm/openmm/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['dc63d7b47c8bb7b169c409cfd63d909ed0ce1ae114d37c627bf7a4231acf488e']
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('Doxygen', '1.9.7'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('SWIG', '4.1.1'),
+    ('CUDA', '11.7.0', '', SYSTEM),
```

Comment:
@smoors are you intentionally sticking to an older CUDA here? the default used in the 2023a gen would be `CUDA/12.1.1`

Reply:
not intentional, thx for catching this.